app/views/layouts/_messages.html.erb

```diff
@@ -1,5 +1,5 @@
 <% flash.each do |name, msg| %>
-  <% if msg.is_a?(String) %>
+  <% if msg.is_a?(String) && ![:timedout].include?(namei.to_sym) %>
```

Comment:
It won't surprise me if the list gets longer as development continues.  It's not a performance bottleneck so my preference is to make it a little easier for the next developer to deal with this code.